docs/tutorials/getting_started/interactions.rst

```diff
@@ -0,0 +1,41 @@
+Accessing the interactions
+==========================
+
+This tutorial will show you briefly how to access the detailed interaction
+results corresponding to the tournament.
+
+As shown in :ref:`getting-started` let us create a tournament except that,
```

Comment:
Maybe something like: "To access the detailed interaction results we create a tournament as usual (see getting-started), setting the `keep_matches=True` parameter to save the interactions."